### PR TITLE
Feat: new ODS major milestones press release

### LINKED ISSUE #252: New press request
### Approved?

Yes

### Title

Cal-ITP and Partners Celebrate Major Milestones in Implementation of Operational Data Standard (ODS) for Transit

### Heading (optional)

_No response_

### Asset URL

https://docs.google.com/document/d/1hA558RcoF8gzOgQdqfdyKn8BGVIYOGnkUmmU28qH6jY/edit

### Tag

GTFS

### Date - Year

2023

### Date - Month

08

### Location

_No response_

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #252

### CODE REVIEW

File: src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+date: "2023-08-22T09:00:00-07:00"
+title: Cal-ITP and Partners Celebrate Major Milestones in Implementation of Operational Data Standard (ODS) for Transit
+heading: Cal-ITP and Partners Celebrate Major Milestones in Implementation of Operational Data Standard (ODS) for Transit
```
[the remark this answers is not in the document]
Comment (esquared415) [reply src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md:4]: Yes, please!

**Review by machikoyasuda — APPROVED** [review]
lgtm!

File: src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thekaveman) [reply src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md:4]: To confirm @esquared415, it should be:

> Cal-ITP and partners celebrate major milestones in implementation of Operational Data Standard (ODS) for transit

[the remark this answers is not in the document]
Comment (esquared415) [reply src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md]: That's correct, @thekaveman. Thank you!

[the remark this answers is not in the document]
Comment (thekaveman) [reply src/_press/cal-itp-partners-celebrate-major-milestone-implementation-ods.md]: It's updated in the preview @esquared415!

**Review by machikoyasuda — APPROVED** [review]

**Review by esquared415 — CHANGES_REQUESTED** [review]
Could you please fix a few minor punctuation things:

- [x] Paragraph 10:
• Delete spaces around the 2 dashes in this graph: 
   " ... convenience and ease—as we’ve done ... "
   " ... one click—another step ... "

• Ensure both dashes in this graph are the same width (should be an em-dash, which is formed by hitting shift + option + - dash).

- [x] Paragraph 12:
• Delete spaces around the 1 dash in this graph: 
Product Manager—Integration

- [ ] About Cal-ITP graph:
• Add closing parenthesis to ([Cal-ITP Benefits](https://www.calitp.org/press/cal-itp-benefits-launch?utm_source=Press+Release&utm_medium=Website&utm_id=ODS+Update))

Thanks!

**Comment by thekaveman** [comment]
@esquared415 fixed those!

**Review by esquared415 — APPROVED** [review]
Looks great—thank you, @thekaveman!

**Pull request merged**